labgrid/driver/powerdriver.py

```diff
@@ -265,7 +274,24 @@ def cycle(self):
 
     @Driver.check_active
     def get(self):
-        return self.pykush.get_port_state(self.port.index)
+        cmd = [
+            "ykushcmd",
+            "-s", f"{self.port.serial}",
+            "-g", f"{self.port.index}"
+        ]
+        res = processwrapper.check_output(self.port.command_prefix + cmd)
+        res = res.decode("utf-8")
+
+        # the example of ykushcmd -g like below:
+        # cmd: ykushcmd -g 1
+        # output: Downstream port 1 is ON/OFF
+        check_str = f"port {self.port.index}"
+        if check_str in res:
+            status = res.strip().split()[-1]
+            if status == "ON":
+                return True
+            else:
+                return False
```

Comment:
```suggestion
        check_str = "Downstream port {port} is {status}"
        if check_str.format(port=self.port.index, status="ON") in res:
            return True
        if check_str.format(port=self.port.index, status="OFF") in res:
            return False
            
        raise ExecutionError(f"Could not find port string in ykushcmd output: {res}")
```
This makes sure we get a return value or an exception in all cases.